website/src/docs/hotchocolate/api-reference/extending-filtering.md

```diff
@@ -284,8 +284,7 @@ A little simplified this is what happens during visitation:
 ```graphql
 {
   users(
-    where: # instance[0] = x # Create SCOPE 1 with parameter x of type User
-    # level[0] = []
+    where: # level[0] = [] # instance[0] = x # Create SCOPE 1 with parameter x of type User
```

Comment:
I guess each `#` represents a new line.

Reply:
From prettier as well, shall I revert it manually? Maybe we should skip .md files with prettier?